### PR TITLE
[ODS-6562] Bring holding bin documents to document repo

### PR DESCRIPTION
**Pull request by mjaksn**

### CODE REVIEW

File: odsApi_versioned_docs/version-7.3/technical-articles/authorizing-requests-using-custom-database-views.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+AssessmentIdentifier and Namespace columns. While these two columns determine
+the query results that must be produced by the custom authorization view
+definition, they don’t necessarily need to come directly from the basis table.
+
+In this example, we are actually interested in filtering Assessments based on
+the presence of the Academic Subject of “Mathematics”. This value is stored in
+the AcademicSubjectDescriptorId column of the AssessmentAcademicSubject child
+table:
+
+![Database Diagram](/img/reference/ods-api/Assessment-AssessmentAcademicSubject-Diagram.webp)
```
Comment (vimayya) [inline odsApi_versioned_docs/version-7.3/technical-articles/authorizing-requests-using-custom-database-views.md:82]: Most of the database diagrams are converted to mermaid diagrams. e.g. https://github.com/Ed-Fi-Alliance-OSS/ed-fi-alliance-oss.github.io/blob/main/odsApi_versioned_docs/version-7.3/platform-dev-guide/configuration/context-based-routing-for-year-specific-ods.md?plain=1#L53. Can you convert this one?

File: static/img/reference/ods-api/view-based-auth-diagram.webp
Comment (vimayya) [inline static/img/reference/ods-api/view-based-auth-diagram.webp:1]: converting to webp increased the size tiny bit here.

**Review by vimayya — COMMENTED** [review]
Holding bin had other two page updates. See what has changed in https://edfi.atlassian.net/wiki/pages/diffpagesbyversion.action?pageId=354091013&selectedPageVersions=3&selectedPageVersions=1 and https://edfi.atlassian.net/wiki/pages/diffpagesbyversion.action?pageId=360251468&selectedPageVersions=1&selectedPageVersions=3

File: odsApi_versioned_docs/version-7.3/technical-articles/authorizing-requests-using-custom-database-views.md (hunk 1 above)
Comment (mjaksn) [reply odsApi_versioned_docs/version-7.3/technical-articles/authorizing-requests-using-custom-database-views.md]: This is now converted to mermaid.

File: static/img/reference/ods-api/view-based-auth-diagram.webp
Comment (mjaksn) [reply static/img/reference/ods-api/view-based-auth-diagram.webp:1]: This is now converted to mermaid.

**Review by vimayya — APPROVED** [review]

**Pull request merged**